Nodejs/Tests/Core/Debugger/Commands/SetBreakpointCommandTests.cs

```diff
@@ -114,8 +147,8 @@ public class SetBreakpointCommandTests {
             Assert.AreEqual(commandId, setBreakpointCommand.Id);
             Assert.AreEqual(
                 string.Format(
-                    "{{\"command\":\"setbreakpoint\",\"seq\":{0},\"type\":\"request\",\"arguments\":{{\"line\":{1},\"column\":{2},\"type\":\"script\",\"target\":\"{3}\",\"ignoreCount\":1}}}}",
-                    commandId, line, column, fileName.Replace(@"\", @"\\")),
+                    "{{\"command\":\"setbreakpoint\",\"seq\":{0},\"type\":\"request\",\"arguments\":{{\"line\":{1},\"column\":{2},\"type\":\"scriptRegExp\",\"target\":\"{3}\",\"ignoreCount\":1}}}}",
+                    commandId, line, column, SetBreakpointCommand.CreateLocalScriptRegExp(fileName).Replace(@"\", @"\\")),
                 setBreakpointCommand.ToString());
         }
```

Comment:
Not to my knowledge. The user experience should be unchanged and I don't see any new failures on the UI tests. Do you have an specific concerns?